README.md

```diff
@@ -225,9 +114,8 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
 
```

Comment:
- [ ] Why dont you remove this comment in your readme file to make it professional since all these comments in the readme template were to show you what to do in each section :blush: